### ISSUE TITLE
NTR: Modifier: ear pain worsened by manipulating tragus or pinna (Ears)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7814]
New term request Modifier: ear pain worsened by manipulating tragus or pinna (segal_210809204335): Helix, lobule

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000173
- label: Ear pain exacerbated by manipulating tragus or pinna
- parents: [Aggravated by (HP:0025285)](https://hpo.jax.org/app/browse/term/HP:0025285)
- pmid: 
- def: Applies to pain of the ear that is worsened, aggravated, or exacerbated by touching or moving the tragus or pinna.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7814
- synonyms:
